CarpetX/src/driver.hxx

```diff
@@ -14,6 +14,7 @@
 #include <AMReX_FluxRegister.H>
 #include <AMReX_Interpolater.H>
 #include <AMReX_MultiFab.H>
+#include <AMReX_FArrayBox.H>
```

Comment:
Please run `clang-format` to sort the included files.

Reply:
Will do. Is there a specific version that I should use?